corehq/apps/data_interfaces/interfaces.py

```diff
@@ -85,12 +87,22 @@ def rows(self):
     @property
     def report_context(self):
         context = super(CaseReassignmentInterface, self).report_context
-        active_users = self.get_all_users_by_domain(user_filter=tuple(HQUserType.all()), simplified=True)
+        if not self.request.can_access_all_locations:
+            accessible_location_ids = (SQLLocation.active_objects.accessible_location_ids(
+                self.request.domain,
+                self.request.couch_user)
+            )
+            accessible_user_ids = user_ids_at_locations(accessible_location_ids)
+            active_users = self.get_all_users_by_domain(user_filter=tuple(HQUserType.all()), simplified=True,
+                                                        user_ids=tuple(accessible_user_ids))
+        else:
+            active_users = self.get_all_users_by_domain(user_filter=tuple(HQUserType.all()), simplified=True)
+            context.update(groups=[dict(ownerid=group.get_id, name=group.name, type="group")
+                                   for group in self.all_case_sharing_groups],
+                           )
         context.update(
             users=[dict(ownerid=user.user_id, name=user.username_in_report, type="user")
                    for user in active_users],
-            groups=[dict(ownerid=group.get_id, name=group.name, type="group")
-                    for group in self.all_case_sharing_groups],
```

Comment:
Oh, nothing, just commenting that this uses a `dict` constructor instead of a `dict` literal:
```python
dict(ownerid=group.get_id, name=group.name, type="group")
# vs
{'ownerid': group.get_id, 'name': group.name, 'type': "group"}
```
(Totally fine to leave as-is, it's just unusual code I don't see often)